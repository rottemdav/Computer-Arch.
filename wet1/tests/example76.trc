8 3 13 0 global_history local_tables using_share_mid
0x04ad28 T 0xe6839168
0x2034c N 0x850d4
0xddc T 0x6a1a9b0
0x65793d98 T 0xbe8250
0x341c3c T 0x8c358
0x9578 T 0x7ea9c
0x341c3c T 0x8c358
0x2034c T 0x850d4
0xb1c N 0x984888
0x04ad28 N 0xe6839168
0x2034c T 0x850d4
0xabfd4 T 0x0910
0x483e773c N 0x1530c
0x1ed91ad8 T 0x01b8
0x65793d98 T 0xbe8250
0x2034c T 0x850d4
0x8dcdb4 T 0x82dbfc
0xd70d0 T 0xcc4
0xd70d0 N 0xcc4
0xabc T 0x53d60
0xeb3a5c N 0xe46138
0x51c8e020 T 0x23d3c
0x4da9e18 N 0x51efcc60
0x9b200 T 0x560
0x1ed91ad8 T 0x01b8
0x2034c N 0x850d4
0xb1c N 0x984888
0xb1c N 0x984888
0xeb3a5c N 0xe46138
0x483e773c N 0x1530c
0x08c4c8 T 0x7679d6bc
0x08c4c8 N 0x7679d6bc
0xc307408 N 0xd4f180
0xbb3ec8 N 0xa8d8e38c
0xd84b4 N 0x1f1b74
0xbe47ef4 T 0xa4c0
0x8dcdb4 T 0x82dbfc
0xb1c N 0x984888
0xeb3a5c N 0xe46138
0x51c8e020 T 0x23d3c
0x9578 N 0x7ea9c
0xabc T 0x53d60
0x4da9e18 T 0x51efcc60
0xb1c N 0x984888
0xd88 N 0x6efec
0xd84b4 N 0x1f1b74
0xabc T 0x53d60
0xbe47ef4 T 0xa4c0
0x0dc N 0xbd04
0x08c4c8 N 0x7679d6bc
0xabfd4 N 0x0910
0xeb3a5c N 0xe46138
0x9578 N 0x7ea9c
0xddc N 0x6a1a9b0
0xd84b4 N 0x1f1b74
0x65793d98 T 0xbe8250
0xddc T 0x6a1a9b0
0xb1c T 0x984888
0xabc T 0x53d60
0xd88 T 0x6efec
0xbe47ef4 N 0xa4c0
0xddc N 0x6a1a9b0
0xabc N 0x53d60
0xb36ef50 T 0x00c8
0xc307408 N 0xd4f180
0xb1c N 0x984888
0xb1c T 0x984888
0xddc N 0x6a1a9b0
0xabc N 0x53d60