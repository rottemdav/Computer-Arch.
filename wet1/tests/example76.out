0x4ad28 N 0x4ad2c
0x2034c N 0x20350
0xddc N 0xde0
0x65793d98 N 0x65793d9c
0x341c3c N 0x341c40
0x9578 N 0x957c
0x341c3c N 0x341c40
0x2034c N 0x20350
0xb1c N 0xb20
0x4ad28 N 0x4ad2c
0x2034c N 0x20350
0xabfd4 N 0xabfd8
0x483e773c N 0x483e7740
0x1ed91ad8 N 0x1ed91adc
0x65793d98 N 0x65793d9c
0x2034c N 0x20350
0x8dcdb4 N 0x8dcdb8
0xd70d0 N 0xd70d4
0xd70d0 N 0xd70d4
0xabc N 0xac0
0xeb3a5c N 0xeb3a60
0x51c8e020 N 0x51c8e024
0x4da9e18 N 0x4da9e1c
0x9b200 N 0x9b204
0x1ed91ad8 N 0x1ed91adc
0x2034c N 0x20350
0xb1c N 0xb20
0xb1c N 0xb20
0xeb3a5c N 0xeb3a60
0x483e773c N 0x483e7740
0x8c4c8 N 0x8c4cc
0x8c4c8 N 0x8c4cc
0xc307408 N 0xc30740c
0xbb3ec8 N 0xbb3ecc
0xd84b4 N 0xd84b8
0xbe47ef4 N 0xbe47ef8
0x8dcdb4 N 0x8dcdb8
0xb1c N 0xb20
0xeb3a5c N 0xeb3a60
0x51c8e020 N 0x51c8e024
0x9578 N 0x957c
0xabc N 0xac0
0x4da9e18 N 0x4da9e1c
0xb1c N 0xb20
0xd88 N 0xd8c
0xd84b4 N 0xd84b8
0xabc N 0xac0
0xbe47ef4 N 0xbe47ef8
0xdc N 0xe0
0x8c4c8 N 0x8c4cc
0xabfd4 N 0xabfd8
0xeb3a5c N 0xeb3a60
0x9578 N 0x957c
0xddc N 0xde0
0xd84b4 N 0xd84b8
0x65793d98 N 0x65793d9c
0xddc N 0xde0
0xb1c N 0xb20
0xabc N 0xac0
0xd88 N 0xd8c
0xbe47ef4 N 0xbe47ef8
0xddc N 0xde0
0xabc N 0xac0
0xb36ef50 N 0xb36ef54
0xc307408 N 0xc30740c
0xb1c N 0xb20
0xb1c N 0xb20
0xddc N 0xde0
0xabc N 0xac0
flush_num: 33, br_num: 69, size: 483b
